SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -418,6 +418,14 @@ void USpatialConnectionManager::SetupConnectionConfigFromURL(const FURL& URL, co
 		DevAuthConfig.MetaData = URL.GetOption(*SpatialConstants::URL_METADATA_OPTION, TEXT(""));
 		DevAuthConfig.WorkerType = SpatialWorkerType;
 	}
+	else if (URL.HasOption(TEXT("customLocator")))
```

Comment:
if we specify a custom locator it will now only work with the locator flow, not the dev auth flow. Not sure if that is the way to go, the changes before looked better imo (I was in meetings till now and couldn't take a closer look at it). Were there some issues with it?

Reply:
discussed it with @mattyoung-improbable offline, thought that isolating this change is the best approach for now as it's only used internally and we're not sure yet whether allowing any locator url is the way to go. Until we have a better understanding of the need for arbitrary locator urls it's probably safest to add it under `customLocator` for now. As custom devAuth urls aren't used (yet) I left it out, but can add something like `customDevAuth` as well